pythonforandroid/toolchain.py

```diff
@@ -588,6 +668,25 @@ def warn_on_deprecated_args(self, args):
         Print warning messages for any deprecated arguments that were passed.
         """
 
+        # Output warning if setup.py is present and neither --ignore-setup-py
+        # nor --use-setup-py was specified.
+        if getattr(args, "private", None) is not None and \
+                (os.path.exists(os.path.join(args.private, "setup.py")) or
+                 os.path.exists(os.path.join(args.private, "pyproject.toml"))
+                ):
+            if not getattr(args, "use_setup_py", False) and \
+                    not getattr(args, "ignore_setup_py", False):
+                warning("  **** FUTURE BEHAVIOR CHANGE WARNING ****")
+                warning("Your project appears to contain a setup.py file.")
+                warning("Currently, these are ignored by default.")
+                warning("This will CHANGE in an upcoming version!")
```

Comment:
Haha, I guess I did suggest that it will change in the future :joy: of course if most of you think that's a bad idea I'll just take it out. But my suggestion would be, just for the sake of having even easier defaults, that eventually this is changed such that `--use-setup-py` is the default